fabio/edfimage.py

```diff
@@ -625,16 +715,29 @@ def check_header(header=None):
             new[toAscii(key, ";{}")] = toAscii(value, ";{}")
         return new
 
+    # PB38k20190607: _read_header_block reads a single header and
+    # interpretes immediately the EDF_ specification keys.
+    # These values (header_size, binary_size) must be defined
+    # at the beginning of the header and are interpreted and
+    # returned as additional return values
+    # For the moment no change, only EDF_HeaderSize is interpreted.
     @staticmethod
```

Comment:
Comments are useless, please complete the docsting so that this content is properly indexed and the documentation properly populated.

Reply:
When replying 4 values I would advice you to use a NamedTupple:
from collection import namedtupple and define a ReturnType
ReturnType = namedtuple("ReturnType", "header_block header_size, binary_size")